usr/share/webapp-manager/firefox/profile/chrome/userChrome.css

```diff
@@ -1,3 +1,7 @@
 #nav-bar, #identity-box, #tabbrowser-tabs, #TabsToolbar {
-	visibility: collapse !important;
+	visibility: collapse;
 }
+
+#nav-bar * {
```

Comment:
This is targetting all of the children of nav-bar, and their children, and so-on recursively. I tried doing "#nav-bar > *" instead, but the result was weird - attempting to add an extension resulted in flickering similar to permission pop-ups before my changes